src/components/tx/security/shared/SecurityWarnings/styles.module.css

```diff
@@ -0,0 +1,64 @@
+.hint {
+  padding: var(--space-2);
+  border: unset;
+}
+
+.hint :global .MuiAlert-icon {
+  padding: 0;
+}
+
+.hint :global .MuiAlert-message {
+  padding: 0;
+}
+
+.warning {
+  padding: var(--space-2) var(--space-4);
+  background-color: unset;
+}
+
+.warning :global .MuiAlert-message {
+  padding: 0;
+}
+
+.checkbox {
+  margin: var(--space-2) 0;
+}
+
+.checkbox :global .MuiCheckbox-root {
+  padding: 0 12px;
+}
+
+.verdictBox {
+  border: none;
+  border-radius: 6px;
+  border-bottom: 1px solid var(--color-border-light);
+}
+
+.verdictBox :global .MuiAccordionSummary-content {
+  margin-bottom: 0;
+}
```

Comment:
This causes a small visual bug where its not centered anymore

<img width="577" alt="Screenshot 2023-06-15 at 15 17 19" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/3783eb1e-ee95-49e7-a5a0-d01041ec872a">